### PR TITLE
I upload the Chinese traslation for the latest TAIGA version.

### PR DESCRIPTION
**Pull request by metalluster**
That's my first time to upload files in the github, hope I did something right.
Uhm, I've communicated with the mod owner ZKaface.

### CODE REVIEW

File: src/main/resources/assets/taiga/lang/zh_CN.lang [hunk 1]
```diff
@@ -1,210 +1,304 @@
...
-material.prometheum.name=幻金
-material.rubium.name=鲁宾
+material.basalt.name=玄武岩
+material.dilithium.name=双锂
 material.tiberium.name=泰伯利亚
-material.arcanite.name=奥金
-material.eternite.name=伊特尼
-material.mythril.name=秘银
+material.aurorium.name=奥罗拉
+material.prometheum.name=幻金
```
Comment (3TUSK) [inline src/main/resources/assets/taiga/lang/zh_CN.lang]: https://en.wikipedia.org/wiki/Promethium ?

File: src/main/resources/assets/taiga/lang/zh_CN.lang [hunk 2]
```diff
@@ -1,210 +1,304 @@
...
-material.bysmuid.name=拜斯麦
-material.nucleum.name=辐光合金
-material.terramite.name=大地合金
+material.proxii.name=普罗克希
+material.tritonite.name=漩金
+material.ignitz.name=焰晶
+material.imperomite.name=帝金
 material.solarium.name=阳光合金
-material.lumixyl.name=流明
+material.nihilite.name=虚空金属
```
Comment (3TUSK) [inline src/main/resources/assets/taiga/lang/zh_CN.lang]: 确定“虚空金属”这个名字不会和神秘时代的那个混淆...？翻译本身是没有问题的。

**Comment by 3TUSK** [comment]
Nice work!

File: src/main/resources/assets/taiga/lang/zh_CN.lang (hunk 2 above)
Comment (metalluster) [inline src/main/resources/assets/taiga/lang/zh_CN.lang]: 那么虚无金属？其实知道这个有混淆，但并不是我翻译的。实在不想改了，教程里也好，搬运贴也好，都已经这么写了，改了不仅是工作量大的事。

File: src/main/resources/assets/taiga/lang/zh_CN.lang (hunk 1 above)
Comment (metalluster) [inline src/main/resources/assets/taiga/lang/zh_CN.lang]: So sorry that I didn't realize what the word prometheum means.
('Cause that can't be translated by mainstream online translators but promithium can.
I'll correct the two wrong places of the lang file at the same time asap!

File: src/main/resources/assets/taiga/lang/zh_CN.lang (hunk 2 above)
Comment (3TUSK) [inline src/main/resources/assets/taiga/lang/zh_CN.lang]: 好吧 - -||

File: src/main/resources/assets/taiga/lang/zh_CN.lang (hunk 1 above)
Comment (3TUSK) [inline src/main/resources/assets/taiga/lang/zh_CN.lang]: You might want to confirm with author though. That one is still my guess...

**Pull request closed without merging**